rotkehlchen/tests/unit/test_accounting.py

```diff
@@ -502,3 +502,69 @@ def test_accounting_works_for_empty_history(accountant):
     )
     assert FVal(result['overview']['general_trade_profit_loss']).is_close('0')
     assert FVal(result['overview']['total_taxable_profit_loss']).is_close('0')
+
+
+# asset_movements_list partially copied from
+# rotkehlchen/tests/integration/test_end_to_end_tax_report.py
```

Comment:
It would probalby make sense in the future to move those into a utils or data file under the tests directory so that both tests can access it and data is not duplicated